governance/votes/Executive Vote - August 2, 2023.md

```diff
@@ -0,0 +1,141 @@
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Dai Savings Rate will be increased, as detailed below.
+- The Spark Protocol Dai Direct Deposit Module Maximum Debt Ceiling will be increased, as detailed below.
+- Changes will be made to the Harbor Trade Credit (RWA004-A) vault, as detailed below.
+- The New Silver (RWA002-A) `doc` parameter will be updated, as detailed below.
+- A total of **134.1 MKR** will be distributed to 10 AVC members, as detailed below.
+- A DAO Resolution for the Monetalis Clydesdale (RWA007-A) structure will be approved, as detailed below.
+- **2,000,000 DAI** will be distributed for Launch Project funding, as detailed below.
+- A Spark Protocol proxy spell will be triggered, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Enhanced Dai Savings Rate (DSR) Activation
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmcTRPLx), the Stability Scope has been amended to include an [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-). To achieve this, the DSR will be updated if this executive proposal passes.
+
+- Increase the DSR by 4.81% from 3.19% to **8%**.
+
+Please review the forum [thread](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405) for more information about this change.
+
+### Spark Dai Direct Deposit Module (D3M) Parameter Change
+
+Per this successful governance [poll](https://vote.makerdao.com/polling/QmSLj3HS), the Spark Protocol D3M will be updated if this executive proposal passes.
+
+- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 180 million DAI from 20 million DAI to **200 million DAI**.
+
+The [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and the [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) will remain unchanged at 20 million DAI and 8 hours respectively.
```

Comment:
Not sure you really need this? I guess it doesn't hurt.

Reply:
I'm anticipating being asked to include it by technical, agree it's largely redundant